index.js

```diff
@@ -67,7 +68,8 @@ var supportedTargets = [
   {runtime: 'electron', target: '2.0.0', abi: '57', lts: false},
   {runtime: 'electron', target: '3.0.0', abi: '64', lts: false},
   {runtime: 'electron', target: '4.0.0', abi: '64', lts: false},
-  {runtime: 'electron', target: '4.0.4', abi: '69', lts: false}
+  {runtime: 'electron', target: '4.0.4', abi: '69', lts: false},
+  {runtime: 'electron', target: '5.0.0', abi: '70', lts: false}
```

Comment:
Are you sure it should be `70`? I believe it uses node v12, which has `NODE_MODULE_VERSION = 72` as above.

Reply:
> I believe it uses node v12, which has `NODE_MODULE_VERSION = 72` as above.

There's ABI differences between the two that need to be handled downstream, even if they report the same version. See https://github.com/electron/electron/pull/16687 for the back story about why Electron needs to get its own ABIs. 

Here's the ABI version that the Electron team use when building Electron 5:

https://github.com/electron/electron/blob/7514372e7959d7e44286797a2f3688382d6a90b3/build/args/all.gn#L5-L6

```
# Registry of NMVs --> https://github.com/nodejs/node/blob/master/doc/abi_version_registry.json
node_module_version = 70
```

And here's the quick start demo reporting the same version:

<img width="830" src="https://user-images.githubusercontent.com/359239/56682813-b9d52700-66a2-11e9-97ad-10a8df267f59.png">

Reply:
Thanks for the explanation. Looking at the links and screenshot it appears that you are correct. 

However, there still seems to be a problem with your fork/PR. Here's wh

```sh
$ git clone git@github.com:shiftkey/node-abi.git
cd node-abi
npm install
node
```

```js
const abi = require('./index.js');
abi.getAbi('5.0.0', 'electron')
Error: Could not detect abi for version 5.0.0 and runtime electron.  Updating "node-abi" might help solve this issue if it is a new release of electron
    at getAbi (/home/user/path/node_modules/node-abi/index.js:31:9)
    at Object.exports.getAbi (/home/user/path/node_modules/node-abi/index.js:113:18)
```

Reply:
For whatever reason, removing the `5.0.0-beta.0` entry from the `futureTargets` array resolves this.

```diff
 var futureTargets = [
-  {runtime: 'electron', target: '5.0.0-beta.0', abi: '68', lts: false}
 ]
```

Reply:
@jacobq I'll do that now. Thanks for noticing that!

Reply:
Fixed in d2f77bf (which should also fix that failing test).